src/Integration.Vsix/AsmRef_Integration.Vsix_Baseline_WithStrongNames.txt

```diff
@@ -258,9 +258,10 @@ Referenced assemblies:
 - 'SonarLint.VisualStudio.Core, Version=4.28.0.0, Culture=neutral, PublicKeyToken=c5b62af9de6d7244'
 - 'SonarLint.VisualStudio.IssueVisualization, Version=4.28.0.0, Culture=neutral, PublicKeyToken=c5b62af9de6d7244'
 - 'System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'
+- 'System.ComponentModel.Composition, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'
 - 'System.Core, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'
 - 'System.Xaml, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'
-# Number of references: 16
+# Number of references: 18
```

Comment:
Note that the previous `Number of references` was shown as 16. I've checked and there are 18. The 16 is presumably due to a merge issue. It's not important and wasn't picked up by the assembly ref checker because it was only a comment (the assembly ref checker checks the detail of the actual asm refs).